plugins/MobileAppType/lang/en.json

```diff
@@ -0,0 +1,7 @@
+{
+  "MobileAppType": {
+    "MobileApp": "Mobile App",
+    "MobileApps": "Mobile Apps",
+    "MobileAppDescription": "A native mobile app for eg iOS, Android, .."
```

Comment:
Suggested description: `A native mobile app for iOS, Android or Titanium. Measure user interactions with one of our SDKs.` ?